src/components/settings/layout.tsx

```
@@ -1,137 +1,252 @@
+          setError("An unexpected error occurred. Please try again later.")
+        }
+      } finally {
+        setLoading(false)
+      }
+    }
+
+    fetchData()
+  }, [guildId])
+
+  function isAxiosError(error: any): error is { response?: { data?: { message?: string } } } {
+    return error && error.response
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)